user_manual/_templates/versions.html

```diff
@@ -0,0 +1,40 @@
+{% if READTHEDOCS %}
```

Comment:
What is this file about exactly?

Reply:
This is the template to have this:

![582-456-max](https://user-images.githubusercontent.com/1863453/89033236-a6e5e380-d336-11ea-9d73-9690553f41b4.png)

As recommended [here](https://github.com/readthedocs/sphinx_rtd_theme/issues/857).

You can see it live on the [test website](https://doc-nextcloud-fr.indie.host/en/) I pushed.

Reply:
Thanks! :)
I was wondering why we needed a custom template for this.
Maybe we should add a comment to explain this is not part of th official rtd theme yet so we have to use our own?